package-lock.json

```diff
@@ -0,0 +1,7291 @@
+{
```

Comment:
Stray `package-lock.json` file, probably `npm` was accidentally used instead of `pnpm`